test/unit/mallctl.c

```diff
@@ -883,5 +1075,7 @@ main(void) {
 	    test_arenas_lookup,
 	    test_stats_arenas,
 	    test_hooks,
-	    test_hooks_exhaustion);
+	    test_hooks_exhaustion,
+	    test_utilization_query,
```

Comment:
The util query tests deserve their own tests I feel. Let's move them to a `util_query.c` in a follow up diff.